src/staticdata_utils.c

```diff
@@ -837,7 +838,47 @@ static void jl_insert_methods(jl_array_t *list)
         assert(!meth->is_for_opaque_closure);
         jl_methtable_t *mt = jl_method_get_table(meth);
         assert((jl_value_t*)mt != jl_nothing);
-        jl_method_table_insert(mt, meth, NULL);
+        jl_typemap_entry_t *entry = jl_method_table_add(mt, meth, NULL);
+        jl_array_ptr_set(list, i, entry);
+    }
+}
+
+static void jl_activate_methods(jl_array_t *list, jl_array_t *internal, size_t world)
```

Comment:
Can we use more descriptive names for `internal` and `list`? Maybe `methodtables` and `methods`?